src/main/java/core/basesyntax/FootballMatch.java

```diff
@@ -15,6 +15,11 @@
      * если не угадал ничего - возвращать 0.</p>
      */
     public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+
+        int winBet = (result1 == bet1 && result2 == bet2) ? 2
+                : (result2 > result1 && bet2 > bet1 || result1 > result2 && bet1 > bet2
+                || result2 == result1 && bet2 == bet1) ? 1
+                : 0;
+        return winBet;
```

Comment:
do we necessarily need to create new variable "winBet", or can we return result without it?

Reply:
I deleted winBet.